### PR TITLE
Cleanup System.*Exception tests

### PR DESCRIPTION
**Pull request by hughbe**
- Rename and cleanup tests

Contributes to #6286
Cherry picked from #5490

### CODE REVIEW

File: src/System.Runtime/tests/System/FieldAccessException.cs [hunk 1]
```diff
@@ -3,40 +3,38 @@
...
 
 using System;
-
 using Xunit;
 
 public static class FieldAccessExceptionTests
 {
-    internal const int COR_E_METHODACCESS = unchecked((int)0x80131510);
-    internal const int COR_E_FIELDACCESS = unchecked((int)0x80131507);
+    private const int FieldAccessHResult = unchecked((int)0x80131507);
```
Comment (stephentoub) [inline src/System.Runtime/tests/System/FieldAccessException.cs]: Please revert this.  The name that was being used matches the native name and should be kept.

File: src/System.Runtime/tests/System/MethodAccessException.cs [hunk 2]
```diff
@@ -3,40 +3,38 @@
...
 
 using System;
-
 using Xunit;
 
 public static class MethodAccessExceptionTests
 {
-    internal const int COR_E_METHODACCESS = unchecked((int)0x80131510);
-    internal const int COR_E_FIELDACCESS = unchecked((int)0x80131507);
+    private const int MethodAccessHResult = unchecked((int)0x80131510);
```
Comment (stephentoub) [inline src/System.Runtime/tests/System/MethodAccessException.cs]: Same comment.  Please revert this.

File: src/System.Runtime/tests/System/MissingFieldException.cs [hunk 3]
```diff
@@ -3,40 +3,38 @@
...
 
 using System;
-
 using Xunit;
 
 public static class MissingFieldExceptionTests
 {
-    internal const int COR_E_MISSINGMETHOD = unchecked((int)0x80131513);
-    internal const int COR_E_MISSINGFIELD = unchecked((int)0x80131511);
+    private const int MissingFieldHResult = unchecked((int)0x80131511);
```
Comment (stephentoub) [inline src/System.Runtime/tests/System/MissingFieldException.cs]: Revert.

**Comment by stephentoub** [comment]
LGTM other than the requested reversions.

**Comment by hughbe** [comment]
Done

**Comment by hughbe** [comment]
Test Innerloop CentOS7.1 Debug Build and Test
Test Innerloop CentOS7.1 Release Build and Test
Test Innerloop Windows_NT Debug Build and Test

**Comment by hughbe** [comment]
Test Innerloop Windows_NT Debug Build and Test

**Comment by stephentoub** [comment]
Test Innerloop Windows_NT Debug Build and Test

**Pull request merged**